src/Fixer/Basic/BracesFixer.php

```diff
@@ -30,23 +30,32 @@
  */
 final class BracesFixer extends AbstractFixer implements ConfigurableFixerInterface, WhitespacesAwareFixerInterface
 {
+    const NEXT_LINE = 'next';
+
+    const SAME_LINE = 'same';
+
+    private $supportedBracePositions = array(self::NEXT_LINE, self::SAME_LINE);
+
     /**
      * @var array
      */
     private $configuration;
 
     private static $defaultConfiguration = array(
         'allow_single_line_closure' => false,
+        'classy_constructs' => self::NEXT_LINE,
     );
 
     /**
-     * @param array<string, bool>|null $configuration
+     * @param array|null $configuration
+     *
+     * @throws InvalidFixerConfigurationException
      */
     public function configure(array $configuration = null)
     {
-        if (null === $configuration) {
-            $this->configuration = self::$defaultConfiguration;
+        $this->configuration = self::$defaultConfiguration;
```

Comment:
no. as docs stated, null will reset the configuration, array will override it, not patch it on top of default values.